src/authentication/Revoked.jsx

```diff
@@ -11,8 +11,9 @@ class Revoked extends Component {
   }
 
   async logBackIn() {
```

Comment:
Un-break loging back in again with the new cozy client, missed that one last time.